libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -356,6 +356,13 @@ void AP_Vehicle::setup()
 #endif
 
 #if AP_SERIALMANAGER_ENABLED
+#if HAL_WITH_IO_MCU
+    if (BoardConfig.io_enabled()) {
+        // If IO is enabled, we need to set the protocol to None
+        // as the IO MCU UART is handled by the AP_IOMCU class
```

Comment:
What happens if we *don't* do this?

Reply:
there's a typo that it needs to say IOMCU not None

Reply:
comment removed altogether, as its pretty clear from the code what is happening.